### DISCUSSION TITLE
Will .NET MAUI provide pixel-perfect renderers ?

### DISCUSSION THREAD

**Discussion by redradist** [dotnet/maui#582] [Q&A]
I am interested in pixel-perfect renderers, will .NET MAUI provide possibility to have the same look on all platforms via say SkiaSharp ?

**Comment by jsuarezruiz** [comment]
We are evaluating this option. We actually have a prototype here: https://github.com/dotnet/GraphicsControls
It allows drawing each control using the drawing APIs of each platform (Example: iOS uses CoreGraphics etc) or using SkiaSharp.

We are reviewing and validating the idea as a possible option and in order to use it for Visual (allow to use Material, Fluent etc and have exactly the same rendering pixel by pixel in all the platforms).

Any feedback is welcome: https://github.com/dotnet/GraphicsControls/issues/1
